### PR TITLE
Fix links-preview error for the messegers

### PR DESCRIPTION
**Pull request by LarysaShashuk**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -111,6 +111,18 @@ async function createConfig() {
                         name: 'twitter:image',
                         content: 'img/docs-preview.jpg'
                     },
+                    {
+                        property: "og:title",
+                        content: 'iCure Documentation | Get started, build your Medical Device App or EHR with iCure'
+                    },
+                    {
+                        property: 'og:description',
+                        content: 'iCure is a trustable service ensuring the privacy of your patient medical data thanks to end-to-end encryption, that empowers your users to decide who can access their data.'
```
Comment (lisatr2812) [inline docusaurus.config.js:120]:
```suggestion
                        content: 'iCure is a trustable service ensuring the privacy of your patient medical data thanks to end-to-end encryption and empowering your users to decide who can access their data.'
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -111,6 +111,18 @@ async function createConfig() {
...
+                    },
+                    {
+                        name: 'description',
+                        content: 'iCure is a trustable service ensuring the privacy of your patient medical data thanks to end-to-end encryption, that empowers your users to decide who can access their data.'
```
Comment (lisatr2812) [inline docusaurus.config.js:124]:
```suggestion
                        content: 'iCure is a trustable service ensuring the privacy of your patient medical data thanks to end-to-end encryption and empowering your users to decide who can access their data.'
```

File: docusaurus.config.js [hunk 3]
```diff
@@ -111,6 +111,18 @@ async function createConfig() {
                         name: 'twitter:image',
                         content: 'img/docs-preview.jpg'
                     },
+                    {
+                        property: "og:title",
+                        content: 'iCure Documentation | Get started, build your Medical Device App or EHR with iCure'
```
Comment (lisatr2812) [inline docusaurus.config.js:116]:
```suggestion
                        content: 'iCure Documentation | Start to build your Medical Device or EHR App with iCure'
```

**Review by lisatr2812 — CHANGES_REQUESTED** [review]
Small typos to update

**Review by aduchate — APPROVED** [review]

**Review by aduchate — APPROVED** [review]

**Review by lisatr2812 — APPROVED** [review]

**Pull request merged**